fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1915,6 +1933,22 @@ public static Builder newBuilder() {
             return new Builder();
         }
 
+        /**
+         * Create an indexing policy builder from existing object.
+         * @return an {@link IndexingPolicy} builder, defaulted to the existing object's values.
+         */
+        @Nonnull
+        public Builder toBuilder() {
+            return newBuilder()
```

Comment:
One other approach that I've seen work for this is for the `Builder` to have a private constructor that takes an `IndexingPolicy` and then initializes all of the variables appropriately. If done right, it can be structured so that the compiler will give you an error if you add a field and forget to pass it along. But this is also probably fine if you don't want to make that change.